src/doom/DoomMain.java

```diff
@@ -1523,7 +924,22 @@ protected void GenerateTitle() {
             title.append(VERSION%100);
             title.append("                           ");
             break;
-
+        case freedoom1:
+            title.append("                            ");
+            title.append("FreeDoom: Phase 1          v");
+            title.append(VERSION/100);
+            title.append(".");
+            title.append(VERSION%100);
+            title.append("                           ");
+            break;
+        case freedoom2:
+            title.append("                            ");
+            title.append("FreeDoom: Phase 2          v");
+            title.append(VERSION/100);
+            title.append(".");
+            title.append(VERSION%100);
+            title.append("                           ");
+            break;
```

Comment:
Okay, I totally agree with that. Added to my checklist.
By the way, Freedoom support is far from full: it is very basic at the moment. We do not use either Freedoom unique messages, or monster names how they should appear in Finale.
I figured that out when comparing to the reference.